xl2times/transforms.py

```diff
@@ -772,39 +787,52 @@ def process_time_periods_table(table: datatypes.EmbeddedXlTable):
     return [process_time_periods_table(table) for table in tables]
 
 
-def generate_all_regions(
+def process_regions(
     config: datatypes.Config,
     tables: List[datatypes.EmbeddedXlTable],
+    model: datatypes.TimesModel,
 ) -> List[datatypes.EmbeddedXlTable]:
     """
     Include IMPEXP and MINRNW together with the user-defined regions in the AllRegions set
```

Comment:
Just noticing that this sentence is missing a `.` at the end -- when docstring is extracted it will be concatenated with the next line with a space ` `.  If you make more edits to this PR, might be worth adding a `.`.